api/rest/route/routeregister.go

```diff
@@ -3,12 +3,14 @@ package route
 import (
 	"github.com/gofiber/fiber/v2"
 	"github.com/hashicorp/raft"
-	"nubedb/consensus/fsm"
+	"nubedb/cluster/consensus/fsm"
 	"nubedb/internal/app"
+	"nubedb/internal/config"
 )
 
 // ApiCtx is a simple struct to include a collection of tools that a route could need to operate, for example a DB.
 type ApiCtx struct {
+	Config     config.Config
```

Comment:
Should config just be passed like that?

Reply:
I don't see the problem for the api rest to have the config parameters, since it also has consensus and fsm pointers